app/jobs/course/material/zip_download_job.rb

```diff
@@ -1,5 +1,6 @@
 class Course::Material::ZipDownloadJob < ApplicationJob
   include TrackableJob
+  include PublishFile
```

Comment:
Maybe we should call this send file? https://www.nginx.com/resources/wiki/start/topics/examples/xsendfile/